doc/source/whatsnew/v1.5.1.rst

```diff
@@ -8,7 +8,7 @@ including other versions of pandas.
 
 {{ header }}
 
-.. ---------------------------------------------------------------------------
+.. --------------------------------------------------------------------------
```

Comment:
Nit: Could you undo this change? (This file will generally receive backports which we do not want to do for this file)